.meepctl-repocfg.yaml

```diff
@@ -42,6 +44,25 @@ repo:
       ca: self-signed
       # lets-encrypt production server (true) or staging server (false)
       le-server-prod: false
+      
+    auth:
+      # session encryption key
+      session-key: my-secret-key
+      # OAuth redirect URI
+      redirect-uri: https://my-platform-fqdn/platform-ctrl/v1/authorize
+      # Github OAuth configuration
+      github:
+        # Client app identifier
+        client-id: my-client-id
+        # Oatuh app secret
+        secret: my-secret
+      # Gitlab OAuth configuration
+      gitlab:
+        # Client app identifier
+        client-id: my-client-id
+        # Oatuh app secret
+        secret: my-secret
+
```

Comment:
`client-id` and `secret` should point to a K8s secret name which holds the secret key

Reply:
Agreed. This will be addressed in an upcoming PR.